src/SeaIceThermodynamics/SeaIceThermodynamics.jl

```diff
@@ -0,0 +1,143 @@
+module SeaIceThermodynamics
+
+export SlabSeaIceThermodynamics, 
+       PhaseTransitions,
+       MeltingConstrainedFluxBalance,
+       PrescribedTemperature,
+       RadiativeEmission,
+       ConductiveFlux,
+       FluxFunction
+
+#####
+##### A bit of thermodynamics to start the day
+#####
+
+abstract type AbstractSeaIceThermodynamics end
```

Comment:
Why do we need an abstract type?

Reply:
Probably not necessary at the moment that we have only one thermodynamically model. I can remove it now and we can reintroduce it when we have more models

Reply:
also we only need an abstract type if we use it for dispatch somehow

Reply:
was this addressed?